json/src/lib.rs

```diff
@@ -1140,6 +1140,39 @@ pub enum PubKeyOrAddress<'a> {
     PubKey(&'a PublicKey),
 }
 
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, Debug)]
+#[serde(untagged)]
+pub enum ScanTxoutRequest {
+    Single(String),
+    Extended {
+        dest: String,
+        range: u64,
+    }
+}
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, Debug)]
+pub struct ScanUtxoResult {
+    pub success: bool,
+    pub txouts: u64,
+    pub height: u64,
+    pub bestblock: bitcoin::BlockHash,
```

Comment:
I'm not quite sure about the naming convention for `bestblock`. Names are usually converted to snake_case, but `GetTxOutResult` uses `bestblock`, while `GetBlockchainInfoResult` uses `best_block_hash`.

Reply:
fixed